openedx/features/course_experience/views/course_home_messages.py

```diff
@@ -114,7 +114,7 @@ def register_course_home_messages(request, course, user_access, course_start_dat
             CourseHomeMessages.register_info_message(
                 request,
                 Text(_(
-                    "{add_reminder_open_tag}Don't forget to add a calendar reminder!{add_reminder_close_tag}."
+                    "{add_reminder_open_tag}Don't forget to add a calendar reminder!{add_reminder_close_tag}"
```

Comment:
Slid this in here so we didn't need to make another story - can remove and post in separate PR if you would like.